package.json

```diff
@@ -82,7 +82,7 @@
   "husky": {
     "hooks": {
       "prepare-commit-msg": "exec < /dev/tty && git cz --hook || true",
-      "pre-push": "yarn test"
+      "pre-push": "yarn lint && yarn test && cd example && yarn lint"
```

Comment:
You're running `yarn lint` twice

Reply:
the first one is for the lib itself & the second one is for the `example`.

`yarn lint` without `cd` into the example was not running it.

Reply:
I have refactor the `lint` script to run `eslint` on both the lib & the project, and kept the pre-push neater.